lte/gateway/c/core/oai/tasks/amf/amf_security_mode_control.cpp

```diff
@@ -305,11 +300,11 @@ int amf_proc_security_mode_control(
 
   OAILOG_DEBUG(
       LOG_NAS_AMF,
-      "Initiating security mode control procedure, "
-      "KSI = %d\n",
+      "initiating security mode control procedure, "
```

Comment:
for this and the other logs in general as well, why were these changed to start with lower case? in my opinion for cleanliness these all should start with capital letter, to follow the OAI codebase logging format

Reply:
comment addressed.

I have changed to lowercase all log statements to make unform. not aware OAI codebase guidelines.
all log statements changed as suggested, starts with capital letter.